CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ Changes since v2.16
   - The external title is shown to applicants in the application and internal name used throughout the administration.
   - The API supports the old style for now.
 - Answers to conditional fields that are not visible are no longer stored by REMS. The API accepts answers for invisible fields but drops them. The UI does not send answers to invisible fields. (#2574)
+- The form editor UI was reworked to look less cluttered. Many inputs are now hidden behind "Show" buttons by default.
```

Comment:
Should this link to the issue we have? #1899

Reply:
I guess I didn't add it originally because this doesn't resolve the ticket, but I added the link now. It makes sense.